### PR TITLE
Add Support travis-ci

### PR DESCRIPTION
**Pull request by sctnightcore**
issues #2789 

- [ ] login -> char -> map check

EX..
https://github.com/Openkore-File/openkore/pulls

### CODE REVIEW

File: src/functions.pl [hunk 1]
```diff
@@ -428,6 +428,7 @@ sub promptFirstTimeInformation {
 
 sub processServerSettings {
 	my $filename = shift;
+	exit(0) if ($Settings::travis_ci);
```
Comment (sctnightcore) [inline src/functions.pl]: it cannot check error in pServer_servertype

Comment (sctnightcore) [reply src/functions.pl]: I think we need check with 
login -> char -> map -> wait 5 sec -> exit

**Comment by Nipodemos** [comment]
that's a nice ideia

maybe someday we can use this to test new servertypes that you send :smile:

File: src/functions.pl (hunk 1 above)
Comment (Nipodemos) [reply src/functions.pl]: why wait 5 seconds?

**Comment by Nipodemos** [comment]
i think we should merge this.

**Comment by sctnightcore** [comment]
@Nipodemos  It need wait someone add travis-ci 

i cannot add it ...

**Comment by koooge** [comment]
Hi, CI is a good idea to ensure reliability!
How about [Github Actions](https://github.com/features/actions) instead of 3rd party CI SaaS like travisci?

**Comment by alisonrag** [comment]
we already use github actions

**Pull request closed without merging**